chart/templates/patch_serviceaccount.yaml

```diff
@@ -0,0 +1,64 @@
+{{- include "_config.load" $ }}
+{{- if gt (len .Values.kube.image_pull_secrets) 0 }}
```

Comment:
This is fine, but could be simplified (because an empty list is `false`) to:

```
{{- if .Values.kube.image_pull_secrets }}
```

Reply:
confirmed